draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -409,8 +404,9 @@ A clash between an existing member zone's name and an incoming member zone's nam
 
 ## Member zone removal {#zoneremoval}
 
-When a member zone is removed from a specific catalog zone, an authoritative server MUST NOT remove the zone and associated state data if the zone was not configured from that specific catalog zone.
-Only when the zone was configured from a specific catalog zone, and the zone is removed as a member from that specific catalog zone, the zone and associated state (such as zone data and DNSSEC keys) MUST be removed.
+When a member zone is removed from a specific catalog zone, a consumer MUST NOT remove the zone and associated state data if the zone was not configured from that specific catalog zone.
+Only when the zone was configured from a specific catalog zone, and the zone is removed as a member from that specific catalog zone, the zone and associated state (such as zone data and DNSSEC keys) MUST be removed from the consumer.
```

Comment:
In [this comment](https://github.com/NLnetLabs/draft-toorop-dnsop-dns-catalog-zones/issues/60#issuecomment-1411892469) (commit: fdd8d79bcd3adfafb707b84a6561fb55dbb41425) and the ones following, the word "removed" was actually replaced by "served". This was also the whole point of issue #60.

Later, other parts of the sentence were changed. Was the revert to "removed" intentional?

Reply:
> Later, other parts of the sentence were changed. Was the revert to "removed" intentional?

It was. We addressed the point eventually by adding another line:

```
   Consumer operators may consider to temporarily archive associated state to facilitate mistake recovery.
```

Reply:
Sorry, my bad -- I had not drawn the connection between these two sentences. Entschuldigung!